prow/cmd/gcsupload/BUILD.bazel

```diff
@@ -15,6 +15,18 @@ go_library(
     ],
 )
 
+load("@io_bazel_rules_docker//go:image.bzl", "go_image")
+
+go_image(
+    name = "image",
+    base = "@alpine-base//image",
+    embed = [":go_default_library"],
+    goarch = "amd64",
+    goos = "linux",
+    pure = "on",
+    visibility = ["//visibility:public"],
+)
```

Comment:
`gcsupload` did not have a Bazel defined image rule. I also notice that the Dockerfile for it drops the binary at `/`, so again, if the binary *must* be located at `/` then I will revert this change.

Reply:
This one does not have to be at `/`